Content.Server/Atmos/Consoles/AtmosMonitoringConsoleSystem.cs

```diff
@@ -0,0 +1,541 @@
+
+        if (!TryComp<GridAtmosphereComponent>(gridUid, out var atmosphere))
+            return;
+
+        // The grid must have a NavMapComponent to visualize the map in the UI
+        EnsureComp<NavMapComponent>(gridUid);
+
+        // Gathering data to be send to the client
+        var atmosNetworks = new List<AtmosMonitoringConsoleEntry>();
+        var query = AllEntityQuery<GasPipeSensorComponent, TransformComponent>();
+
+        while (query.MoveNext(out var ent, out var entSensor, out var entXform))
+        {
+            if (entXform?.GridUid != xform.GridUid)
+                continue;
+
+            if (!entXform.Anchored)
+                continue;
+
+            var entry = CreateAtmosMonitoringConsoleEntry(ent, entXform);
+
+            if (entry != null)
+                atmosNetworks.Add(entry.Value);
+        }
+
+        // Set the UI state
+        _userInterfaceSystem.SetUiState(uid, AtmosMonitoringConsoleUiKey.Key,
+            new AtmosMonitoringConsoleBoundInterfaceState(atmosNetworks.ToArray()));
+    }
+
+    private AtmosMonitoringConsoleEntry? CreateAtmosMonitoringConsoleEntry(EntityUid uid, TransformComponent xform)
+    {
+        AtmosMonitoringConsoleEntry? entry = null;
+
+        var netEnt = GetNetEntity(uid);
+        var name = MetaData(uid).EntityName;
+        var address = string.Empty;
+
+        if (xform.GridUid == null)
+            return null;
+
+        if (!TryGettingFirstPipeNode(uid, out var pipeNode, out var netId) ||
+            pipeNode == null ||
+            netId == null)
+            return null;
+
+        // Name the entity based on its label, if available
+        if (TryComp<LabelComponent>(uid, out var label) && label.CurrentLabel != null)
+            name = label.CurrentLabel;
+
+        // Otherwise use its base name and network address
+        else if (TryComp<DeviceNetworkComponent>(uid, out var deviceNet))
+            address = deviceNet.Address;
+
+        // Entry for unpowered devices 
+        if (TryComp<ApcPowerReceiverComponent>(uid, out var apcPowerReceiver) && !apcPowerReceiver.Powered)
+        {
+            entry = new AtmosMonitoringConsoleEntry(netEnt, GetNetCoordinates(xform.Coordinates), netId.Value, name, address)
+            {
+                IsPowered = false
+            };
+
+            return entry;
+        }
+
+        // Entry for powered devices 
+        var gasData = new Dictionary<Gas, float>();
+        var isAirPresent = pipeNode.Air.TotalMoles > 0;
+
+        if (isAirPresent)
+        {
+            foreach (var gas in Enum.GetValues<Gas>())
+            {
+                if (pipeNode.Air[(int)gas] > 0)
+                    gasData.Add(gas, pipeNode.Air[(int)gas] / pipeNode.Air.TotalMoles);
+            }
+        }
+
+        entry = new AtmosMonitoringConsoleEntry(netEnt, GetNetCoordinates(xform.Coordinates), netId.Value, name, address)
+        {
+            TemperatureData = isAirPresent ? pipeNode.Air.Temperature : 0f,
```

Comment:
Showing 0 Kelvin for vacuum does not really make sense from a physics perspective.
Sending 0 as default data is fine, but it should not be displayed on the monitor.
![grafik](https://github.com/user-attachments/assets/a12b3167-4781-4a1e-a798-e04eaa41786a)

Reply:
Updated the display so if the total number of moles is very small, it reads a temperature of 'N/A'